metabase/metabase
# replace search bar with command palette toggle

Closes https://github.com/metabase/metabase/issues/42346

### Description
PR swaps out the search bar for a button in the `AppBar` component when we are not embedded. Button should toggle the command palette. Tests were updated to either
1) Test in embedding mode when we were testing functionality of the floating search results or
2) Perform the search via the command palette

### How to verify
1. click the Search button on the top of the screen. It should open the command palette
2. should also work in mobile view
3. test that embedding still shows original search bar

### Demo
![chrome_bFHMwf0qUl](https://github.com/metabase/metabase/assets/1328979/0d900f03-6616-42ea-ada3-ee0b1d171a33)


### Checklist
